code/parachain/frame/pallet-multihop-xcm-ibc/Cargo.toml

```diff
@@ -20,8 +20,9 @@ package = "parity-scale-codec"
 version = "3.0.0"
 
 [dependencies]
-bech32-no_std = { workspace = true, default-features = false, features = ["strict"] }
+bech32-no_std = { version = "0.7.3", default-features = false, features = ["strict"] }
```

Comment:
why not to change in workspace? we started to use it only for cosmos thing just now